indico/modules/events/papers/forms.py

```diff
@@ -40,3 +49,7 @@ def __init__(self, *args, **kwargs):
             del self.content_reviewers
         if not self.event.cfp.layout_reviewing_enabled:
             del self.layout_reviewers
+
+
+class PaperCompetencesForm(IndicoForm):
```

Comment:
Just use `IndicoForm` directly in the `type()` call.

    form_class = type(b'PaperCompetencesForm', (IndicoForm,), {})